File: Java/OPSJLib/src/ops/Event.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package ops;

import java.util.Observable;

/**
 *
 * @author Anton Gravestam
 */
public class Event extends Observable
{
    
    public void fireEvent(Object args)
    {
        setChanged();
        notifyObservers(args);
    }
    public void fireEvent()
    {
        setChanged();
        notifyObservers();
    }
}

File: trunk/C#/Ops/Event.cs
///////////////////////////////////////////////////////////
//  Event.cs
//  Implementation of the Class Event
//  Created on:      12-nov-2011 09:25:30
//  Author:
///////////////////////////////////////////////////////////

using System;

namespace Ops 
{
	public class Event : Observable 
    {
        public void FireEvent(object args)
        {
            NotifyObservers(args);
        }
        public void FireEvent()
        {
            NotifyObservers(null);
        }

	}

}